<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>851</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>Quản lý giá vé</string>
    </property>
    <widget class="QLineEdit" name="txtMaVe">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>ID Gía Vé</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLamMoi">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>370</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Làm Mới</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tblGiaVe">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>481</width>
       <height>241</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Loại Vé</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Loại Xe</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Giá Vé</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="txtGiaVe">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Giá vé</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbLoaiVe">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Luot</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Thang</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nam</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnSua">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>360</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Cập Nhật</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbLoaiXe">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Oto</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XeMay</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XeDap</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnXoa">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>360</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Xóa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnThem">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>360</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Bổ sung</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTuKhoa">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Tìm Kiếm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTimKiem">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>30</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Tìm Kiếm</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnThoat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
